Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -17,6 +17,7 @@ public class CRTClientEngine: HttpClientEngine {
         private let windowSize: Int
         private let maxConnectionsPerEndpoint: Int
         private var connectionPools: [Endpoint: HTTPClientConnectionManager] = [:]
+        private var http2ConnectionPools: [Endpoint: HTTP2StreamManager] = [:]
```

Comment:
Should this be a separate map of connection pools from `connectionPools`, or should connection type be a property of `Endpoint` so that all connections (HTTP 1.1 & 2.0) can be stored together?